### ISSUE TITLE
Add example to typescript usage guide

### ISSUE THREAD

**Issue by kentcdodds** [kentcdodds/glamorous-website#223]
**Problem Description:**

[Something](https://github.com/paypal/glamorous/issues/211) was confusing

**Suggested Solution:**

Make it less confusing by including @luke-john's example from that issue in [the typescript docs](https://github.com/kentcdodds/glamorous-website/blob/master/pages/api/content/typescript.md) :smile:

**Comment by luke-john** [comment]
I've also got an [in progress introduction to glamorous with typescript blog post](https://gist.github.com/luke-john/2335f748cbcf4acb8fcb5357b00fccef) which I'll hopefully get time to finish sometime this week.  Some of the examples from that will likely be handy for this section.

Note: the typings on that blog post are based on the typings for vnext of glamorous from https://github.com/paypal/glamorous/pull/230. I also started writing that at the same time as the vnext typings so it still has the shape of some things incorrect (ie, theme as the second argument to style functions).
